client/src/components/Container.jsx

```diff
@@ -51,15 +71,19 @@ class Container extends React.Component {
       });
   }
 
-  getAverageStars(storeId) {
-    console.log(this.state.reviews);
+  getAverageStars(reviews) {
+    console.log(`getAverageStores hit`);
     var average;
```

Comment:
always declare undefined variables as `null`

Reply:
Actually in this specific case, you should declare average below

Reply:
why's it best to declare them as null?